product_docs/docs/pem/8/changing_default_port.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Changing the default port"
+---
+
+The PEM Server is configured after the installation either automatically (in Windows) or manually by running the configuration script (in Linux). While configuring the PEM Server, by default the 8443 port is assigned for the web services. You can change the port as per your requirement after configuration by changing a few parameters in the apache configuration files. The apache configuration files names and locations are platform specific.
+
+## For RHEL and CentOS
+
+```text
+# In the file /etc/httpd/conf.d file, replace the port 8443 to your 
+# choice for the following parameters:
+Listen 8443
+<VirtualHost _default_:8443>
+ServerName localhost:8443
+RewriteRule ^(.*)$ https://%{HTTP_HOST}:8443%{REQUEST_URI} [L,R=301]
+
+# Check if the port of your choice is listed:
+semanage port -l | grep http
+
+# If the port of your choice is not listed than run the below command:
+sudo semanage port -a -t http_port_t -p tcp <port_number>
+
+# Restart the httpd service
+systemctl restart httpd
+```
+
+Where `<port_number>` is the port number of your choice.
+
+!!! Note
+    If semanage is not installed then install using `sudo yum -y install policycoreutils-python` command.
+
+After restarting the httpd services you can access the PEM web interface using your new port. For more details see [Accessing the web interface](/pem/latest/pem_web_interface/).
```

Comment:
Since restarting is a step, you could simplify this by saying:
You can now access the PEM web interface using your port. 

You need a comma after "details."

Reply:
cahnges done